src/auxiliary/mpi.jl

```diff
@@ -102,16 +100,19 @@ recursive_sum_abs2(u::Number) = abs2(u)
 # https://github.com/SciML/RecursiveArrayTools.jl
 # However, what you have is good enough for us for now, so we don't need this 
 # additional dependency at the moment.
-recursive_sum_abs2(u::AbstractArray) = mapreduce(recursive_sum_abs2, +, u; init=zero(eltype(eltype(u))))
+function recursive_sum_abs2(u::AbstractArray)
+    mapreduce(recursive_sum_abs2, +, u; init = zero(eltype(eltype(u))))
+end
 
 recursive_length(u::Number) = length(u)
 recursive_length(u::AbstractArray{<:Number}) = length(u)
 recursive_length(u::AbstractArray{<:AbstractArray}) = sum(recursive_length, u)
-function recursive_length(u::AbstractArray{<:StaticArrays.StaticArray{S, <:Number}}) where {S}
-  prod(StaticArrays.Size(eltype(u))) * length(u)
+function recursive_length(u::AbstractArray{<:StaticArrays.StaticArray{S, <:Number}}) where {
+                                                                                            S
+                                                                                            }
```

Comment:
One downside of JuliaFormatter is that sometimes things like this are produced. The following might be a less ugly alternative:
```suggestion
function recursive_length(u::AbstractArray{<:StaticArrays.StaticArray{S,
                                                                      <:Number}}) where {S}
```

Reply:
If you change it this way, it won't get reformatted again?

Reply:
Your suggestion worked. Fixed in cc268cf72.